audit/src/main/java/org/wildfly/security/audit/FileAuditEndpoint.java

```diff
@@ -70,22 +70,22 @@ public class FileAuditEndpoint implements AuditEndpoint {
     }
 
     void setFile(final File file) throws IOException {
-        boolean ok = false;
+        boolean isSetUp = false;
```

Comment:
@OluchukwuCatherineObi-Njoku Please rename to for example `isFileSet` or `fileSet` to be more specific about what has been set

Reply:
Hello. I made the suggested updates. Thanks

Reply:
> Hello. I made the suggested updates. Thanks

@Skyllarr